test/phylorefs.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Test phyloreferences.
+ */
+
+/* eslint-env mocha */
+
+global.Vue = require('vue');
+
+// Load d3 as a global variable so it can be accessed by both phylotree.js (which
+// needs to add additional objects to it) and phyx (which needs to call it).
+global.d3 = require('d3');
+
+if (!Object.prototype.hasOwnProperty.call(global.d3, 'layout')) {
+  global.d3.layout = {};
+}
+require('../lib/phylotree.js/phylotree.js');
```

Comment:
So this doesn't need to be kept as an object?

Reply:
Nope, it just adds itself to global.d3.layout! I've improved the comments to better document that.